scripts/qa_crowbarsetup.sh

```diff
@@ -1281,6 +1281,11 @@ function onadmin_set_source_variables()
             CLOUDSLE12TESTISO="CLOUD-7-TESTING-$arch*Media1.iso"
             CLOUDLOCALREPOS="SUSE-OpenStack-Cloud-7-devel"
         ;;
+        mitakacloud7)
+            CLOUDSLE12DISTPATH=/ibs/Devel:/Cloud:/7:/Mitaka/images/iso
+            CLOUDSLE12DISTISO="SUSE-OPENSTACK-CLOUD-7-$arch*Media1.iso"
+            CLOUDLOCALREPOS="SUSE-OpenStack-Cloud-7-official"
+        ;;
```

Comment:
I don't think that's enough; you probably need to do changes in all other places where we mention develcloud7, and there are probably other locations where we check if the source starts with develcloud...